.nais/nais-prod.yaml

```diff
@@ -0,0 +1,112 @@
+kind: Application
+apiVersion: nais.io/v1alpha1
+metadata:
+  name: amt-distribusjon
+  namespace: amt
+  labels:
+    team: amt
+spec:
+  image: {{image}}
+  port: 8080
+  prometheus:
+    enabled: true
+    path: /internal/prometheus
+  readiness:
+    path: /internal/health/readiness
+    initialDelay: 20
+  liveness:
+    path: /internal/health/liveness
+    initialDelay: 20
+  startup:
+    path: /internal/health/liveness
+    initialDelay: 30
+    failureThreshold: 45
+    periodSeconds: 5
+  replicas:
+    min: 1
```

Comment:
Det er kanskje ikke så nøye for denne appen, men er det ikke litt greit å ha minimum 2 replicas i prod, så er det vel litt tryggere hvis noe uventet skjer med en av de?